File: PooCC01CNetz/Semana2/Python/Ejercicios3/AlcanceAtributos.py
class AlcanceAtributos:
    def __init__(self, atributoPrivado = None):
        self.atributoPublico = None
        self.__atributoPrivado = atributoPrivado

    def inicializaAP(self, atributoPrivado):
        self.__atributoPrivado = atributoPrivado

    def obtenerAP(self):
        return self.__atributoPrivado

File: PooCC01CNetz/Semana2/Java/Ejercicios3/src/AlcanceAtributos.java
public class AlcanceAtributos {
    // Atributos
    public String atributoPublico;
    private String atributoPrivado;

    public AlcanceAtributos(String atributoPrivado) {
        this.atributoPrivado = atributoPrivado;
    }

    public AlcanceAtributos() {}

    //Inicializa el atributo privado
    public void inicializaAP(String atributoPrivado) {
        this.atributoPrivado = atributoPrivado;
    }

    public String obtenerAP() {
        return atributoPrivado;
    }
}